openmmtools/forces.py

```diff
@@ -669,23 +683,28 @@ class RadiallySymmetricCentroidRestraintForce(RadiallySymmetricRestraintForce,
         The indices of the first group of atoms to restrain.
     restrained_atom_indices2 : iterable of int
         The indices of the second group of atoms to restrain.
+    controlling_parameter_name : str, optional
+        The name of the global parameter controlling the energy function.
+        The default value is 'lambda_restraints'.
 
     Attributes
     ----------
     restraint_parameters
     restrained_atom_indices1
     restrained_atom_indices2
+    controlling_parameter_name
 
     """
 
     def __init__(self, energy_function, restraint_parameters,
-                 restrained_atom_indices1, restrained_atom_indices2):
+                 restrained_atom_indices1, restrained_atom_indices2,
+                 controlling_parameter_name='lambda_restraints'):
```

Comment:
Ah, see, here in this subclass, `controlling_parameter_name` an optional keyword arg, which changes the call from the parent function. We should make it consistent in the base class as well

Reply:
> We should make it consistent in the base class as well

Sounds good! I'll move the default to the base class and use kwargs here. I'll have to keep it buried in `kwargs` in the base class though because Python 2 doesn't allow having both kwargs and keyword arguments.

Reply:
Oh yeah... I forgot about that. Might be best to leave it then and it can be removed in 0.17 when we drop Py2 officially

Reply:
Ah, I just rememberd why I did this. It's because I need to use `controlling_parameter_name` in the subclass so I need the default to be specified there. I wouldn't write the default in two places as it's harder to keep track of (i.e., if a dev changes only one of the two the code may become inconsistent and unintuitive).

Reply:
Also a good point